libimagcounter/src/counter.rs

```
@@ -197,7 +193,7 @@ impl<'a> FromStoreId for Counter<'a> {
     fn from_storeid(store: &Store, id: StoreId) -> Result<Counter> {
         debug!("Loading counter from storeid: '{:?}'", id);
         match store.retrieve(id) {
-            Err(e) => Err(CE::new(CEK::StoreReadError, Some(Box::new(e)))),
+            Err(_) => Err(CEK::StoreReadError.into_error()),
```

Here we hide an error as well, please undo this change.